package.json

```diff
@@ -44,21 +42,23 @@
     "jsdom-global": "^3.0.2",
     "mocha": "^4.1.0",
     "mocha-teamcity-reporter": "^2.4.0",
-    "puppeteer": "^1.3.0",
+    "puppeteer": "^1.11.0",
     "react": "^15.2.0",
     "react-dom": "^15.2.0",
-    "selenium-webdriver": "^4.0.0-alpha.1",
+    "selenium-webdriver": "^3.6.0",
     "typescript": "^2.6.2"
   },
   "peerDependencies": {
-    "@types/puppeteer": "^0.13.9",
-    "@types/selenium-webdriver": "^3.0.8",
+    "@types/puppeteer": "^1.11.1",
     "selenium-webdriver": "^3.6.0",
-    "puppeteer": "^0.13.0",
+    "puppeteer": "^1.11.0",
     "protractor": "^5.4.1",
     "react": "^15.2.0",
     "react-test-utils": "0.0.1",
     "react-dom": "^15.2.0"
   },
-  "dependencies": {}
+  "dependencies": {
+    "protractor": "^5.4.2",
```

Comment:
@GabiGrin - this looks wrong. I’d expect it to be a peerDep

Reply:
true! missed that, will fix and release